src/cli/commands/files/add.js

```diff
@@ -1,11 +1,45 @@
 'use strict'
 
 const Command = require('ronin').Command
-const IPFS = require('../../../core')
+const utils = require('../../utils')
 const debug = require('debug')
 const log = debug('cli:version')
 log.error = debug('cli:version:error')
 const bs58 = require('bs58')
+const fs = require('fs')
+const async = require('async')
+const path = require('path')
+const glob = require('glob')
+
+function checkPath (inPath, recursive) {
+  // This function is to check for the following possible inputs
+  // 1) "." add the cwd but throw error for no recursion flag
+  // 2) "." -r return the cwd
+  // 3) "/some/path" but throw error for no recursion
+  // 4) "/some/path" -r
+  // 5) No path, throw err
+  // 6) filename.type return the cwd + filename
+
+  if (!inPath) {
+    throw new Error('Error: Argument \'path\' is required')
+  }
+
+  var s = fs.statSync(inPath)
+
+  if (s.isDirectory() && recursive === false) {
+    throw new Error('Error: ' + process.cwd() + ' is a directory, use the \'-r\' flag to specify directories')
+  }
+  if (inPath === '.' && recursive === true) {
+    inPath = process.cwd()
+    return inPath
```

Comment:
I mean, since there is the return of inPath in the end of the func and this assignment of inPath is within a if/else clause, you just need the return once